src/Tools/CreateTools/CreateGeomTool.js

```diff
@@ -140,19 +144,24 @@ class CreateGeomTool extends BaseCreateTool {
    * @return {boolean} The return value.
    */
   onPointerDown(event) {
+    // skip if the alt key is held. Allows the camera tool to work
+    if (event.altKey) return
```

Comment:
I think we can one day have all these behaviour shorcuts somewhere centrailized. With that, users could change it to their comodity.